meta-ostro-xt/recipes-devtools/checksec/checksec_1.5-1.bbappend

```diff
@@ -0,0 +1,3 @@
+SRC_URI[md5sum] = "075996be339ab16ad7b94d6de3ee07bd"
+SRC_URI[sha256sum] = "77b8a7fd9393d10def665658a41176ee745d5c7969a4a0f43cefcc8a4cd90947"
```

Comment:
@avalluri we need to get checksec_1.5-1.bb in ostro-os-xt because everybody building the distro without OSTRO_SOURCE_MIRROR_URL gets the failure. The old copy of checksec.sh only exists in our mirror.